google/cloud/baremetalsolution/doc/main.dox

```diff
@@ -41,8 +41,8 @@ which should give you a taste of the Bare Metal Solution API C++ client library
 
 <!-- inject-endpoint-env-vars-start -->
 
-- `GOOGLE_CLOUD_CPP_BARE_METAL_SOLUTION_ENDPOINT=...` changes the default endpoint
-  (baremetalsolution.googleapis.com) used by `BareMetalSolutionConnection`.
+- `GOOGLE_CLOUD_CPP_BARE_METAL_SOLUTION_ENDPOINT=...` changes the default `EndpointOption`,
```

Comment:
I don't think it is accurate to say it changes the default `EndpointOption`.

What about just: "`BLAH_ENDPOINT=...` overrides the `EndpointOption` used by `MakeBlahConnection()`."

Reply:
Fair enough.  I took what you had, but kept the mention of the default value.  PTAL.